packages/core/package.json

```diff
@@ -116,6 +115,14 @@
     "vite-plugin-eslint": "^1.8.1",
     "vitest": "^2.0.3"
   },
+  "peerDependencies": {
```

Comment:
@nperez0111 does this makes sense as a peerdependency even if we're only using a type from this package?